crates/astria-bridge-withdrawer/src/withdrawer/submitter/signer.rs

```diff
@@ -0,0 +1,34 @@
+use std::{
+    fs,
+    path::Path,
+};
+
+use astria_core::crypto::SigningKey;
+use astria_eyre::eyre::{
+    self,
+    eyre,
+};
+use sequencer_client::Address;
+
+pub(super) struct SequencerSigner {
```

Comment:
This type does not in fact sign anything as far as I can see? I would expect there to be a method `SequencerSigner::sign`.